camp/accounts/views.py

```diff
@@ -9,3 +16,34 @@ class ProfileDetailView(LoginRequiredMixin, DetailView):
 
     def get_object(self):
         return self.request.user
+
+
+# Put a create page under /accounts/profile/create, and similar for edit.
+
+
+class MembershipListView(ListView):
```

Comment:
As discussed, we don't currently need a membership list view. You can display the user's memberships on their profile detail page by adding to camp/accounts/templates/account/profile_detail.html

To select the membership data to display, you can use the fact that we currently don't care about supporting multiple games and just display all of the user's memberships. If the user has no memberships, you might try displaying a button to create it instead.